### ISSUE TITLE
Requires "pip install qtstylish" or else the import may fail

### ISSUE THREAD

**Issue by sharpe5** [adamerose/PandasGUI#155]
PLEASE FILL OUT THE TEMPLATE

**Describe the bug**  

As per subject.

I used this on 2021-07-08:

Under Miniconda + Python 3.8, I used this:
```
pip install git+https://github.com/adamerose/pandasgui.git
```

However, in my Python script, when I used `from pandasgui import show`, it could not import due to missing `qtstylish`.

The fix was:

```
pip install qtstylish
```

**Environment**
OS: Windows 10
Python: 3.8 ([Miniconda](https://docs.conda.io/en/latest/miniconda.html))
IDE: PyCharm

**Package versions**  
TO GET ALL RELEVANT PACKAGE VERSIONS, RUN THIS COMMAND IN BASH AND PASTE THE OUTPUT
pip freeze | grep -i "pyqt\|pandasgui\|plotly\|ipython\|jupyter\|notebook"

EXAMPLE OUTPUT

> Note: this is *after* I fixed the issue with `pip install qtstylish`.

```
(py38) S:\>pip freeze | grep -i "pyqt\|pandasgui\|plotly\|ipython\|jupyter\|notebook"
ipython @ file:///D:/bld/ipython_1625028461559/work
ipython-genutils==0.2.0
jupyter-client @ file:///home/conda/feedstock_root/build_artifacts/jupyter_client_1615693636836/work
jupyter-core @ file:///D:/bld/jupyter_core_1612125450516/work
jupyter-server @ file:///home/conda/feedstock_root/build_artifacts/jupyter_server_1624558637735/work
jupyterlab @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_1621368934875/work
jupyterlab-pygments @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_pygments_1601375948261/work
jupyterlab-server @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_server_1622556871679/work
notebook @ file:///home/conda/feedstock_root/build_artifacts/notebook_1621259862661/work
pandasgui @ git+https://github.com/adamerose/pandasgui.git@93487a8c1112f0fe7df4853996c1ae92459fa7be
plotly @ file:///home/conda/feedstock_root/build_artifacts/plotly_1624301569624/work
PyQt5==5.15.4
PyQt5-Qt5==5.15.2
PyQt5-sip==12.9.0
PyQtWebEngine==5.15.4
PyQtWebEngine-Qt5==5.15.2
```

**Comment by jmp247** [comment]
I faced the same issue.  I could not `import show` because modules were missing for win32api and qtstylish.  
I had to `pip install pypiwin32` and `pip install qtstylish`

It was a long journey for me to get pandasGUI installed, probably because I'm on an older windows 8.1 machine.

To make a long story short, other requirements: 
- 64-bit version of python
-  I had to install the correct C++ distribution and components for my Windows machine. (Visual Studio 2017 Installer so that I could install Windows 8.1 SDK and others)

I am very impressed by this project, and surprised that this is so new. No alternative like it from what I can see.

**Comment by loperdeve** [comment]
Same here.
```
python -m pandasgui.gui
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/tmp/venv/lib/python3.8/site-packages/pandasgui/__init__.py", line 15, in <module>
    from pandasgui.gui import show
  File "/tmp/venv/lib/python3.8/site-packages/pandasgui/gui.py", line 20, in <module>
    import qtstylish
ModuleNotFoundError: No module named 'qtstylish'
```


fixed it by
``` pip install qtstylish```
Better add to the requirements :)

**Comment by tadeubas** [comment]
Same here, installed with command: `pip install pandasgui`, trying to execute gives error:
```
from pandasgui import show
from pandasgui.gui import show
from pandasgui.widgets.navigator import Navigator
import win32api
ERROR: Could not find a version that satisfies the requirement win32api (from versions: none)
ERROR: No matching distribution found for win32api
```

The above error was solved using the command: `pip install pywin32`, then  trying to execute gives another error:

```
from pandasgui.gui import show
import qtstylish
```

The above error was solved using the command: `pip install qtstylish`

**Comment by adamerose** [comment]
Should be fixed in 0.2.13
